app/src/main/java/at/bitfire/icsdroid/ui/nav/MainApp.kt

```diff
@@ -94,9 +95,16 @@ fun MainApp(
             entry(Destination.SubscriptionList) {
                 SubscriptionsScreen(
                     requestPermissions,
+                    onAboutRequested = { backStack.add(Destination.Info) },
                     onAddRequested = { backStack.add(Destination.AddSubscription()) }
                 )
             }
+            entry(Destination.Info) {
+                InfoScreen(
+                    compStartupServices,
+                    onNavigateUp = ::goBack
```

Comment:
Should we rename this to `onBackRequested` ?  We already have it in `AddSubscriptionScreen` and it would fit with the other names being `onAboutRequested` etc.